lightning/src/ln/channel.rs

```diff
@@ -6972,15 +6977,24 @@ impl<SP: Deref> Writeable for Channel<SP> where SP::Target: SignerProvider {
 			} else if !pending_outbound_skimmed_fees.is_empty() {
 				pending_outbound_skimmed_fees.push(None);
 			}
+			if htlc.blinding_point.is_some() {
+				if pending_outbound_blinding_points.is_empty() {
+					for _ in 0..idx { pending_outbound_blinding_points.push(None); }
+				}
+				pending_outbound_blinding_points.push(htlc.blinding_point);
+			} else if !pending_outbound_blinding_points.is_empty() {
+				pending_outbound_blinding_points.push(None);
+			}
```

Comment:
Good point, it's not required anymore since I switched these to odd TLVs. Still, it is nice to not write a bunch of `None`s in 99% of cases? 

Matt had some feedback in the past that we should macro-ize ser code like this, since it's also repeated in `Route::write`. Could make a follow-up for that, as another option.